src/constant/index.js

```diff
@@ -221,7 +221,7 @@ const SCOPES = [
   },
   {
     id: "customers:rw",
-    description: "Read/write permission for customers",
+    description: "Read/write permission for existing customers. It allows for creating new customers via the Agent Chat API.",
```

Comment:
@bkubiak

Reply:
"It allows *also*..."?

But I don't insist :)